src/__tests__/starWarsSchema.ts

```diff
@@ -100,7 +98,7 @@ const episodeEnum = new GraphQLEnumType({
  *     secretBackstory: String
  *   }
  */
-const characterInterface = new GraphQLInterfaceType({
+const characterInterface: GraphQLInterfaceType = new GraphQLInterfaceType({
```

Comment:
Why do you need explicit types in places like that?
Is it required by TS?

Reply:
It's required by TS because there are circular usage

Reply:
Ok 👍